packages/pieces/google-slides/src/lib/actions/create-presentation.ts

```diff
@@ -0,0 +1,37 @@
+import { googleSlidesAuth } from "../..";
+import { createAction, Property } from "@activepieces/pieces-framework";
+import { HttpMethod, httpClient, HttpRequest, AuthenticationType } from "@activepieces/pieces-common";
+
+
+export const createPresentation = createAction({
+    auth: googleSlidesAuth,
+    name: "create_presentation",
+    description: "Create a new presentation",
+    displayName: "Create Presentation",
+    props: {
+        title: Property.ShortText({
```

Comment:
Thanks a lot for providing the outline for the project! However, I believe it might not be very helpful if folks can only suggest the title. In my opinion, it's better not to merge it in this way because it could give people the wrong idea.